src/platforms/javascript/guides/nextjs/manual-setup/run-dev.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Running in development
```

Comment:
Suggest moving this page to Configuration
```suggestion
title: Default Mode
```

Reply:
Hmm, I suspect Next.js' development mode is not the same as "default mode"?
The "Capitalized Title Suggestion" is good, that's our [style guideline](https://www.notion.so/sentry/Capitalization-eb9c6f9dd5614cb9b9f56140f06189c4#2af991efc6474c6f94134dcfbaa08034) for titles.

Reply:
No. I think "Development Mode" should do it.